src/main/java/com/hcl/appscan/jenkins/plugin/scanners/DynamicAnalyzer.java

```diff
@@ -323,7 +325,11 @@ public FormValidation doCheckScanFile(@QueryParameter String scanFile) {
 			return FormValidation.ok();
 		}
 
-		public FormValidation doCheckTarget(@QueryParameter String target) {
+		public FormValidation doCheckTarget(@QueryParameter String target,@RelativePath("..") @QueryParameter String credentials, @AncestorInPath ItemGroup<?> context) {
```

Comment:
Rather than having to show an error here, can we remove "Dynamic Analyzer" from the list of scan types if AppScan 360 credentials are selected?